docs/notes/2.23.x.md

```diff
@@ -160,6 +160,18 @@ including `node_modules`. This can be used to inspect the installation, or to en
 
 Pants will output a more helpful error message if there is no `name` field defined in the `package.json` file, or if the `name` field is empty.
 
+Pants now ships with experimental JSX support, including prettier formatting and js testing.
+Enable by adding a combination of backends like such:
+
+```pants.toml
+backend_packages = [
+  "pants.backend.experimental.javascript",
+  "pants.backend.experimental.javascript.lint.prettier",
+  "pants.backend.experimental.jsx",
```

Comment:
No, I have a hard time seeing somebody not using these backends together.

TBH it might be worth to just let JSX be enabled together with js? It's literally just 2 target types difference, you use the same rules & goals.

Reply:
> TBH it might be worth to just let JSX be enabled together with js? It's literally just 2 target types difference, you use the same rules & goals.

Interesting idea! 

Thinking out loud:

- AIUI someone could write JS without a bundler, but cannot write JSX without one... but maybe that's irrelevant because pants' base `pants.backend.experimental.javascript` definitely (and will always) support using bundlers?
- Worst case, we work out they need to be separate and do some sort of a deprecation/move in future.

So, yeah, I'm happy to have them merged and keep things simpler if you are!

Reply:
+1 for merging

Reply:
1. Correct. I believe Pants, at least in a first iteration, shouldn't attempt to validate that your imports under your own project settings (CommonJS, ESM Modules, webpack bundled or w/e) would work at runtime because it is very finicky. Rather it should just allow as many bundler-isms as possible to make building and testing your JS with pants painless.
2. Yea, there's a non-zero chance I'm missing something, especially for "react-native" which I have 0 experience with.